dags/glam_firefox_desktop.py

```diff
@@ -0,0 +1,190 @@
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from operators.task_sensor import ExternalTaskCompletedSensor
+from glam_subdags.generate_query import (
+    generate_and_run_glean_queries,
+    generate_and_run_glean_task,
+)
+from utils.gcp import gke_command
+from functools import partial
+
+default_args = {
+    "owner": "akommasani@mozilla.com",
+    "depends_on_past": False,
+    "start_date": datetime(2020, 2, 19),
+    "email": [
+        "telemetry-alerts@mozilla.com",
+        "akommasani@mozilla.com",
+    ],
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 1,
+    "retry_delay": timedelta(minutes=30),
+}
+
+PROJECT = "moz-fx-data-glam-prod-fca7"
+BUCKET = "moz-fx-data-glam-prod-fca7-etl-data"
+
+# Fenix as a product has a convoluted app_id history. The comments note the
```

Comment:
I think this comment doesn't apply.

Reply:
That makes sense. I will go ahead and merge the PR while working on the changes needed for this comment